lang/cs/.nuget/packages.config

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
XUnit console runner, used to run tests from console, is solution-level package, so it adds itself here.